pvacseq/lib/netmhc_stab.py

```diff
@@ -27,10 +27,11 @@ def main(args_input = sys.argv[1:]):
     jobid_searcher = re.compile(r'<!-- jobid: [0-9a-fA-F]*? status: (queued|active)')
     result_delimiter = re.compile(r'-{20,}')
     fail_searcher = re.compile(r'(Failed run|Problematic input:)')
+    allele_searcher = re.compile(r'^(.*?) : Distance to traning data .*? nearest neighbor (.*?)\)$', re.MULTILINE)
```

Comment:
Yeah it took me forever to figure out why the correctly spelled regex wasn't working